doc/source/getting-started/installation.rst

```diff
@@ -108,10 +110,13 @@ In order to install the package use the following command:
 
 .. note::
       The ``pip`` program will download and install all the required
-      dependencies except `cupy <https://cupy.dev/>`_ which is required for GPU
-      acceleration. Please install `cupy <https://cupy.dev/>`_ by following the
+      dependencies except `cupy <https://cupy.dev/>`_ and
+      `cuQuantum <https://developer.nvidia.com/cuquantum-sdk>`_
+      which are required for GPU acceleration. Please install `cupy <https://cupy.dev/>`_ by following the
```

Comment:
Shall we clarify that our ``cuquantum`` backend is an alternative GPU implementation, so that installing it is not required to have GPU acceleration (in contrast to ``cupy``)?